### PR TITLE
Replace appveyor with github actions on sulu 2.1

### PR DESCRIPTION
**Pull request by luca-rath**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | -
| Related issues/PRs | #5837
| License | MIT
| Documentation PR | -

#### What's in this PR?

Replace appveyor with github actions

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -207,30 +207,30 @@
...
+            "@php bin/adminconsole lint:container --env stage",
+            "@php bin/websiteconsole lint:container --env stage",
+            "@php bin/adminconsole lint:container --env prod",
+            "@php bin/websiteconsole lint:container --env prod"
         ],
         "lint-doctrine": [
-            "bin/adminconsole doctrine:schema:validate --skip-sync",
-            "bin/adminconsole doctrine:ensure-production-settings --env prod"
+            "@php bin/adminconsole doctrine:schema:validate --skip-sync",
+            "@php bin/adminconsole doctrine:ensure-production-settings --env prod"
```
Comment (alexander-schranz) [inline composer.json:233]: Think this is good as `@php` would also use the same php version which composer was run e.g.:

```
php7.3 composer.phar lint-doctrine
```

will use php7.3 and not then the global php version to run the subcommand.

👍 for this change, we should make this consistent in skeleton the bundles.

**Review by alexander-schranz — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (niklasnatter) [reply composer.json:233]: sounds good! anybody interested in updating this in all bundles? 😅

Comment (luca-rath) [reply composer.json:233]: not really 🙈 The next one who does something in a bundle can just change this as well

Comment (luca-rath) [reply composer.json:233]: /cc @sulu/core-team

File: .github/workflows/test-application.yaml [hunk 2]
```diff
@@ -167,3 +167,99 @@ jobs:
...
+              with:
+                  dependency-versions: ${{ matrix.dependency-versions }}
+
+            - name: Bootstrap test environment
+              run: composer bootstrap-test-environment
+
+            - name: Execute test cases
+              run: composer test
+
+            - name: Install javascript dependencies
```
Comment (niklasnatter) [inline .github/workflows/test-application.yaml]: not sure if this is compatible with windows, but we used the `bahmutov/npm-install` action in the other workflow:
https://github.com/sulu/sulu/blob/8f6c63ab7e4e606e3fea04c1d7e1dfc9b738c1b5/.github/workflows/test-application.yaml#L34-L37

**Review by niklasnatter — APPROVED** [review]

**Pull request merged**